src/entities/rollerBlindsHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { RollerBlindHandler } from "../contracts";
+
+import * as util from "./utils";
+
+export const rollerBlindsCommandsHandler =
+  ({ mqttClient, hubs, mqttConfig }: RollerBlindHandler) =>
+  async (data: Buffer) => {
+    const message = data.toString().replace(/\s/g, "");
+
+    const bridge_address = message.substring(1, 4);
+    const motor_address = message.substring(4, 7);
+    const action = message.substring(7, 8);
+    const position = message.substring(8, 11);
```

Comment:
This won't work. The bridge_address/motor_address won't always be 3 characters.

Reply:
this is done